jenkins/opensearch-dashboards/bwc-test.jenkinsfile

```diff
@@ -0,0 +1,143 @@
+lib = library(identifier: "jenkins@20211118", retriever: legacySCM(scm))
+
+pipeline {
+    options {
+        timeout(time: 3, unit: 'HOURS')
+    }
+    agent none
+    environment {
+        BUILD_MANIFEST = "build-manifest.yml"
+        DEFAULT_BUILD_JOB_NAME = "distribution-build-opensearch-dashboards"
+    }
+    parameters {
+        string(
+            name: 'TEST_MANIFEST',
+            description: 'Test manifest under the manifests folder, e.g. 2.0.0/opensearch-dashboards-2.0.0-test.yml.',
+            trim: true
+        )
+        string(
+            name: 'BUILD_MANIFEST_URL',
+            description: 'The build manifest URL, e.g. https://ci.opensearch.org/ci/dbc/distribution-build-opensearch-dashboards/2.0.0/98/linux/x64/builds/opensearch-dashboards/manifest.yml.',
+            trim: true
+        )
+        string(
+            name: 'AGENT_LABEL',
+            description: 'The agent label where the tests should be executed, e.g. Jenkins-Agent-al2-x64-c54xlarge-Docker-Host.',
+            trim: true
+        )
+    }
+    stages {
+        stage('verify-parameters') {
+            agent {
```

Comment:
Agent can't be none here due to `fileExists`

Error message from Jenkins : `Required context class hudson.FilePath is missing Perhaps you forgot to surround the code with a step that provides this, such as: node`